docs/yaml-config.rst

```diff
@@ -0,0 +1,98 @@
+Read the Docs YAML Config
+=========================
+
+Read the Docs now has support for configuring builds with a YAML file.
+The file, 
+`readthedocs.yml`,
+must be in the root directory of your project.
+
+.. warning:: This feature is in a beta state.
+             Please file an Issue if you find anything wrong.
+
+Supported Settings
+------------------
+
+conda
+~~~~~
+
+The `conda` block supports a `file` option:
+
+* file
+
+Default: `None`
+Type: Path (specified from the root of the project)
+
+The file option specified the Conda environment file to use.
+
+.. code-block:: yaml
+
+	conda:
+	    file: environment.yml
+
+.. note:: Conda is only supported via the YAML file.
```

Comment:
They are supported -- Conda is only supported via YAML, not via the web UI, like everything else.